datascience-notebook/tests/controls/conda_spec.rb

```diff
@@ -0,0 +1,27 @@
+title 'Working Conda'
+
+control 'Conda available' do
+  impact 0
+  title 'Conda installer not should be available to use'
+  desc 'The Conda installer is not preferred, but is the only way to install some packages.'
+  tag 'installer'
+  tag 'conda'
+  tag 'broken'
+
+  describe command('conda info') do
+    its('exit_status') { should eq 1 }
```

Comment:
Reading this I think you're recording that conda doesn't work in this build right now. Which is fine. But it's a bit weird that the `Conda available` test passes if conda is not working - as a user running the test I might be confused.

How about simply letting the test fail and we consciously merge it like that, in the knowledge that we intend to find a solution?

Or maybe we might skip or comment out the test, until we've fixed this?

Reply:
Ah, fixed. Pushed an update to reflect it.

I've got some things I'm planning to try shortly.